graph-db/serverless/src/testUtil/verifyGraphDb.js

```diff
@@ -24,3 +24,17 @@ export const verifyGraphDb = async (datasetTitle, docName) => {
   const { value: { id: edgeId } } = record
   expect(edgeId).not.toBe(null)
 }
+
+export const verifyNotExistInGraphDb = async (datasetTitle, docName) => {
+  // verify the dataset vertex with the given title does not exist
+  const dataset = await global.testGremlinConnection
+    .V().has('dataset', 'title', datasetTitle).next()
```

Comment:
in `deleteCmrCollection.js` you have chained calls like this on new lines for every call. like this: 
```
.V()
.has(...)
.next()
```

That is easier to read than having them all on the same line. I'd switch all of the gremlin calls in this file to match the example above with each chained method call being on a new line.